core/src/test/java/org/bitcoinj/core/TransactionTest.java

```diff
@@ -179,11 +182,26 @@ public void testCLTVPaymentChannelTransactionSpending() {
         tx.getInput(0).setSequenceNumber(0);
         tx.setLockTime(time.subtract(BigInteger.ONE).longValue());
         TransactionSignature fromSig =
-                tx.calculateSignature(0, from, outputScript, Transaction.SigHash.SINGLE, false);
+                tx.calculateSignature(
```

Comment:
Just to make the point clear, I cannot see what you changedin TransactionTest.java (if at all), because all I see is reformatting.